### PR TITLE
=doc 10.1.0-RC1 release notes

### PR DESCRIPTION
**Pull request by jrudolph**

### CODE REVIEW

File: docs/src/main/paradox/scala/http/release-notes/10.1.x.md [hunk 1]
```diff
@@ -2,13 +2,82 @@
...
+ * Removing methods that have been deprecated during the life-time of Akka HTTP 10.0.x. For now, methods only deprecated
+   in the last release of Akka HTTP, 10.0.11, are not yet removed to allow for a smooth transition.
+ * The new client pool implementation introduced in 10.0.11 is now the default.
+ * Documentation has now almost completely been consolidated between Scala and Java pages. Hundreds of directive
+   documentation pages have been merged in a tireless effort by [@jonas](https://github.com/jonas),
+   [@jlprat](https://github.com/jlprat), and Akka team's [@raboof](https://github.com/raboof).
+
+### Akka is not an explicit dependency any more / Removal of Akka 2.4 support
+
+Akka HTTP supports Akka 2.5 (and future versions during the life of Akka HTTP 10.1.x). However, using Akka HTTP with Akka 2.5 used to be
```
Comment (ktoso) [inline docs/src/main/paradox/scala/http/release-notes/10.1.x.md]: Akka HTTP 10.0.x has always supported Akka 2.5, while allowing users to still remain on Akka 2.4.x if they choose to do so. Akka HTTP 10.1.x however requires Akka 2.5.x....

File: docs/src/main/paradox/scala/http/release-notes/10.1.x.md [hunk 2]
```diff
@@ -2,13 +2,82 @@
...
 
-### Akka is not an explicit dependency any more
+See the [announcement](https://akka.io/blog/news/2017/12/22/akka-http-10.1.0-RC1-released.html) and
+closed tickets on the [10.1.0-RC1 milestone](https://github.com/akka/akka-http/milestone/26?closed=1).
 
-Akka HTTP supports both running on top of Akka 2.4 and Akka 2.5. However, using Akka HTTP with Akka 2.5 used to be
+10.1.0-RC1 is the first release candidate for the next minor version of Akka HTTP. The most important changes are
+
+ * Removing support for Akka 2.4.x which is at its end-of-life with the end of 2017. In the future, this will allow us
+   to make use of features that only Akka 2.5 supports.
```
Comment (patriknw) [inline docs/src/main/paradox/scala/http/release-notes/10.1.x.md]: good

File: docs/src/main/paradox/scala/http/release-notes/10.1.x.md [hunk 3]
```diff
@@ -2,13 +2,82 @@
...
+ * Removing methods that have been deprecated during the life-time of Akka HTTP 10.0.x. For now, methods only deprecated
+   in the last release of Akka HTTP, 10.0.11, are not yet removed to allow for a smooth transition.
+ * The new client pool implementation introduced in 10.0.11 is now the default.
```
Comment (ktoso) [inline docs/src/main/paradox/scala/http/release-notes/10.1.x.md]: Please try it out and provide feedback during the RC period of this release so we can iron out any problems we may have missed initially.

File: docs/src/main/paradox/scala/http/release-notes/10.1.x.md [hunk 4]
```diff
@@ -2,13 +2,82 @@
...
+ * Documentation has now almost completely been consolidated between Scala and Java pages. Hundreds of directive
+   documentation pages have been merged in a tireless effort by [@jonas](https://github.com/jonas),
+   [@jlprat](https://github.com/jlprat), and Akka team's [@raboof](https://github.com/raboof).
```
Comment (ktoso) [inline docs/src/main/paradox/scala/http/release-notes/10.1.x.md]: 👍

**Comment by raboof** [comment]
late 👍 😄

**Pull request merged**